### PR TITLE
scala 2.12.7 and 2.13.0-M5

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Update Scala stable to 2.12.7 and development to 2.13.0-M5.

### CODE REVIEW

File: Formula/scala.rb [hunk 1]
```diff
@@ -3,46 +3,47 @@ class Scala < Formula
...
 
     depends_on :java => "1.8+"
 
     resource "docs" do
-      url "https://downloads.lightbend.com/scala/2.12.6/scala-docs-2.12.6.txz"
-      mirror "https://www.scala-lang.org/files/archive/scala-docs-2.12.6.txz"
-      sha256 "fbbca33acf7492b08b69eb969ea211186c1d70ad57e4f2a2145a770bd9d9a61f"
+      url "https://downloads.lightbend.com/scala/2.12.7/scala-docs-2.12.7.txz"
+      mirror "https://www.scala-lang.org/files/archive/scala-docs-2.12.7.txz"
+      url "https://downloads.typesafe.com/scala/2.12.7/scala-docs-2.12.7.txz"
```
Comment (commitay) [inline Formula/scala.rb]: `url` shouldn't be used twice here.

Comment (pthariensflame) [reply Formula/scala.rb]: Whoops!  I could have sworn I already fixed that…

Rebasing now.

**Comment by pthariensflame** [comment]
@commitay How does it look now?

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request merged**